models/articles.model.js

```diff
@@ -1,12 +1,20 @@
-const db = require('../db/connection')
+const db = require("../db/connection");
 
 exports.fetchArticles = () => {
-    return db.query('SELECT a.article_id, a.title, a.author, a.topic, a.created_at, a.votes, a.article_img_url, COUNT(c.comment_id) AS comment_count FROM articles AS a LEFT JOIN comments AS c ON a.article_id = c.article_id GROUP BY a.article_id ORDER BY a.created_at DESC;')
-}
+  return db
+    .query(
+      "SELECT a.article_id, a.title, a.author, a.topic, a.created_at, a.votes, a.article_img_url, CAST(COUNT(c.comment_id) AS INT) AS comment_count FROM articles AS a LEFT JOIN comments AS c ON a.article_id = c.article_id GROUP BY a.article_id ORDER BY a.created_at DESC;"
+    )
```

Comment:
I'm not sure this would work as expected, it is quite unclear when reading this query string due the repeated use of AS, there is a clearer way of presenting this string please refer to the notes